### PR TITLE
[Do not merge] Accommodate voting

### PR DESCRIPTION
**Pull request by arjunu**
Testing notes:
- Investor with access to voting INV_ICBM_ETH_M_HAS_KYC_DUP_HAS_NEURO
- Investor without access INV_HAS_EUR_HAS_KYC
- Issuer without access ISSUER_PREVIEW
- Issuer with access to voting ISSUER_PAYOUT

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -47,10 +47,10 @@
     "test:e2e": "./e2e/setup-e2e.sh",
     "test:e2e:chrome": "./e2e/setup-docker-e2e.sh",
     "test:e2e:vis-reg": "./e2e/setup-vis-reg-e2e.sh",
-    "test:e2e:cypress:vm": "NODE_ENV=production cypress run --browser chrome --headless --env NF_TAGS=\"$(yarn run --silent show-tags)\" --config testFiles=\"**/*.spec.e2e.ts\" --record --parallel --group localNodeTests --ci-build-id $CIRCLE_BUILD_NUM$(echo node)",
```
Comment (arjunu) [inline packages/web/package.json:50]: only tests with `voting,e2e.ts` and `spec.voting.e2e.io.ts` are run

File: .circleci/config.yml [hunk 2]
```diff
@@ -393,8 +393,8 @@ jobs:
             - run:
                 command: yarn workspace @neufund/web build:prod
                 no_output_timeout: 10m
-            - run:
-                command: yarn workspace @neufund/web test:e2e:vis-reg
+#            - run:
+#                command: yarn workspace @neufund/web test:e2e:vis-reg
```
Comment (arjunu) [inline .circleci/config.yml:397]: disable vis reg as it will fail

File: packages/web/app/modules/accommodate-voting/reducer.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const initialState: TAccommodateVotingState = {
+  showCannotLoginModal: false,
+};
+
+export const accommodateVotingReducer: AppReducer<
+  TAccommodateVotingState,
+  typeof accommodateVotingActions
+> = (state = initialState, action): DeepReadonly<TAccommodateVotingState> => {
+  switch (action.type) {
+    case accommodateVotingActions.showCannotLoginModal.getType():
```
Comment (Moejoe90) [inline packages/web/app/modules/accommodate-voting/reducer.ts:19]: Does it ever stop showing? 
As here we only set the state to true

**Review by Moejoe90 — COMMENTED** [review]
One hanging question

File: packages/web/app/modules/accommodate-voting/reducer.ts (hunk 3 above)
Comment (arjunu) [reply packages/web/app/modules/accommodate-voting/reducer.ts:19]: User can't hide it, there's a logout button which the user if forced to click

**Review by Moejoe90 — DISMISSED** [review]

**Review by desfero — DISMISSED** [review]
Looks good. But we definitely should not merge that into master IMO and just keep that on a separate branch

**Review by Moejoe90 — APPROVED** [review]

File: packages/web/app/modules/accommodate-voting/reducer.ts (hunk 3 above)
Comment (andywalton) [reply packages/web/app/modules/accommodate-voting/reducer.ts:19]: @arjunu -> in this case, can we hide the email modal when showing the `showCannotLoginModal` when a user successfully recovers their account but we can't allow them to log in?

**Comment by desfero** [comment]
Voting would be done on a normal platform. Closing this one but let's keep the brach alive for a while

**Pull request closed without merging**